wordpress-mobile/WordPress-Android
# Feature/stock photos upload

Resolves #7312 - adds uploading to the stock photos feature. 

To test:

- Open the media browser
- Tap to add media
- Choose "Free Photos Library"
- Perform a search
- Select one or more photos then tap "Add"

Selected photos should be uploaded and then reflected in the media browser.

![android-stock-media](https://user-images.githubusercontent.com/3903757/37661419-24c29344-2c2b-11e8-8901-640483170784.gif)
